cmd/root.go

```diff
@@ -49,6 +50,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&walletFile, "wallet", "", "wallet file (default is wallet.json)")
 	rootCmd.PersistentFlags().StringVar(&walletClientID, "wallet_client_id", "", "wallet client_id")
 	rootCmd.PersistentFlags().StringVar(&walletClientKey, "wallet_client_key", "", "wallet client_key")
+	rootCmd.PersistentFlags().Int64Var(&nonce, "withNonce", 0, "nonce that will be used in transaction (default is 0)")
```

Comment:
Isn't nonce mandatory? I think user should not be allowed to run cli without nonce

Reply:
nonce can be optionally set with zboxcli, if not it will be queried by gosdk